components/LessonCard.tsx

```diff
@@ -9,23 +9,38 @@ type Props = {
   lessonCount: number
   hourCount: string | number
   description: string
+  currentState?: string
 }
 
 const LessonCard: React.FC<Props> = props => {
   return (
     <div className="card shadow-sm mt-3">
       <div className="card-body p-2">
         <div className="row no-gutters">
+          {props.currentState === 'completed' && (
+            <span className="badge badge-pill text-white position-absolute lesson-card__container_completed">
+              <img
+                className="lesson-card__completed-icon mr-1"
+                width="24"
+                height="24"
+                src="/curriculumAssets/icons/check-circle.svg"
+                alt="icon-complete"
+              />
```

Comment:
Instead of using `svg` icons in our own repo, I think we should use a generic icon library instead so we don't have to maintain our own SVGs. 

I used https://github.com/feathericons/react-feather

To make it look like this: https://songdev.c0d3.com/curriculum

My react code:
```
import { CheckCircle } from 'react-feather'
... 
<CheckCircle style={{height: '15px'}} className="mr-1"/>
<span className="mr-1">COMPLETED</span>
```

css:
```
.lesson-card__container_completed {
  ...
  line-height: 15px;
}
```